libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1749,7 +1752,7 @@ void ExpressionCompiler::appendExternalFunctionCall(
 
 	if (_functionType.gasSet())
 		m_context << dupInstruction(m_context.baseToCurrentStackOffset(gasStackPos));
-	else if (m_context.experimentalFeatureActive(ExperimentalFeature::V050))
+	else if (m_context.evmVersion().canOverchargeGasForCall())
 		// Send all gas (requires tangerine whistle EVM)
```

Comment:
How about actually adding tanginewhistle as a level?

Reply:
We can do that. It was between homestead and byzantium, right?

Reply:
Yep.

Reply:
Then I'll also add spurious dragon.

Reply:
That could be used for the code size limit and the exp repricing check if we implement it.